helpers/oss.md

```diff
@@ -0,0 +1,45 @@
+# OSS(Object Storage Service)
+
+[QOR OSS](https://github.com/qor/oss) provides common interface to operate files in cloud storage, ftp, filesystem etc.
+
+## Usage
+
+Currently, [QOR OSS](https://github.com/qor/oss) only support file system and S3. But you can easily implement your own storage by the interface.
```

Comment:
> But you can easily implement your own storage by the interface.

But you can easily implement your own storage *strategies* by *implementing* the interface.